homeassistant/components/paperless_ngx/coordinator.py

```diff
@@ -0,0 +1,107 @@
+"""Paperless-ngx Status coordinator."""
+
+from __future__ import annotations
+
+from datetime import timedelta
+
+from pypaperless import Paperless
+from pypaperless.exceptions import (
+    InitializationError,
+    PaperlessConnectionError,
+    PaperlessForbiddenError,
+    PaperlessInactiveOrDeletedError,
+    PaperlessInvalidTokenError,
+)
+from pypaperless.models import Statistic
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_API_KEY, CONF_URL
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryError, ConfigEntryNotReady
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
+
+from .const import DOMAIN, LOGGER
+
+type PaperlessConfigEntry = ConfigEntry[PaperlessCoordinator]
+
+UPDATE_INTERVAL = 120
+
+
+class PaperlessCoordinator(DataUpdateCoordinator[Statistic]):
+    """Coordinator to manage Paperless-ngx statistic updates."""
```

Comment:
```suggestion
    """Coordinator to manage Paperless-ngx statistic updates."""

    config_entry: PaperlessConfigEntry
```
This avoids the mypy things